srcpkgs/sublime-music/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'sublime-music'
+pkgname=sublime-music
+version=0.12.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-flit_core python3-build python3-installer python3-Sphinx"
+makedepends="python3-devel python3-pip"
+depends="python3 python3-bleach python3-bottle python3-json5 python3-dateutil python3-gobject python3-Levenshtein python3-mpv python3-peewee python3-chromecast python3-requests python3-semver python3-thefuzz"
+checkdepends="python3-pytest"
+short_desc="Native Subsonic/Airsonic/sonic client for Linux built with Python/GTK+"
+maintainer="El Bachir <bachiralfa@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://sublimemusic.app"
+changelog="https://github.com/sublime-music/sublime-music/blob/master/CHANGELOG.rst"
+distfiles="https://github.com/sublime-music/sublime-music/archive/refs/tags/v${version}.tar.gz"
+checksum=faaf102fd790ecea3d966c9d6e305567c734a1df30eaa6bbdedf93d863bc7435
+
+do_check() {
+	pytest
```

Comment:
this is what the default does, no need to override it

Reply:
Okay.